presto-docs/src/main/sphinx/sql/execute.rst

```diff
@@ -0,0 +1,48 @@
+.. _execute:
+
+=======
+EXECUTE
+=======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    EXECUTE statement_name [USING parameter1[, parameter2, ...]]
```

Comment:
Brackets should have surrounding spaces in the grammar (see SELECT for an example). Write it list this:
```
EXECUTE statement_name [ USING value [, ...] ]